Changelog.md

```diff
@@ -14,16 +14,22 @@
 
 - [#541]: Deserialize specially named `$text` enum variant in [externally tagged]
   enums from textual content
+- [#520]: Add methods `BytesText::inplace_trim_start` and `BytesText::inplace_trim_end`
+  to trim leading and trailing spaces from text events
 
 ### Bug Fixes
 
 - [#537]: Restore ability to deserialize attributes that represents XML namespace
   mappings (`xmlns:xxx`) that was broken since [#490]
+- [#520]: Merge consequent (delimited only by comments and processing instructions)
```

Comment:
If this only applies to the serde deserializer, that should probably be noted here.

Reply:
Also, if this is something that isn't practical to handle in the low-level API for the time being, then it would be good to document the limitation.  And if it may be possible to do in the future (perhaps requiring API changes to do so) then we should clone #474.

I appreciate the detailed writeup of the issue, by the way.

Reply:
I clearly indicated that this applies only to the deserializer. The documentation thing probably solved by the last commit in this branch ([16fca58](https://github.com/tafia/quick-xml/pull/520/commits/16fca5813961c348f36eabfb380fa92d398d0bbb))?